docgen/docgen.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	google_protobuf "github.com/golang/protobuf/protoc-gen-go/descriptor"
+	plugin_go "github.com/golang/protobuf/protoc-gen-go/plugin"
+
+	"github.com/gunk/gunk/docgen/extract"
+	pg "github.com/gunk/gunk/plugin"
+)
+
+func main() {
+	if err := pg.RunPlugin(new(plugin)); err != nil {
+		log.Fatal(err)
+	}
+}
+
+type plugin struct{}
```

Comment:
what about calling this `docgen` or `docPlugin`? then you don't have to rename the import